zerver/webhooks/jira/view.py

```diff
@@ -346,7 +346,7 @@ def api_jira_webhook(
 ) -> HttpResponse:
 
     event = get_event_type(payload)
-    if event in IGNORED_EVENTS:
+    if event is None or event in IGNORED_EVENTS:
         return json_success()
```

Comment:
I don't think this is the right fix; if we wanted to make a silent success in this case, we should remove the `event is not None` check just after.  But I think we want this to be logged as unsupported.  The right fix would probably instead to do `content_func = None` on the first line of this function, but we need to pass an `event` to that function.  

We might also want to figure out why `get_event_type` is returning `None` here.  (But also we should probably raise some sort of exception for an event we can't parse).